lib/perl/Genome/InstrumentData/Command/Import/Basic_fails.t

```diff
@@ -22,82 +25,97 @@ my $library = Genome::Library->create(
-#FIXME can this error be retrieved?
-#$error = $fail->error_message;
-#is($error, 'Failed to get suffix from source file! blah', 'Correct error meassage');
 
-$fail = Genome::InstrumentData::Command::Import::Basic->create(
-    analysis_project => $analysis_project,
-    library => $library,
-    source_files => \@source_files,
-    import_source_name => 'broad',
-    instrument_data_properties => [qw/ sequencing_platform=solexa lane= flow_cell_id=XXXXXX /],
+throws_ok(
+    sub{
+        Genome::InstrumentData::Command::Import::Basic->execute(
+            analysis_project => $analysis_project,
+            library => $library,
+            source_files => [ $source_file ],
+            import_source_name => 'broad',
+            instrument_data_properties => [qw/ sequencing_platform=solexa lane= flow_cell_id=XXXXXX /],
+        );
+    },
+    qr#Failed to parse with instrument data property label/value! lane=#,
+    'Fails w/ invalid instrument_data_properties',
 );
-ok(!$fail->execute, 'Fails w/ invalid instrument_data_properties');
-is($fail->error_message, 'Failed to process instrument data properties!', 'Correct error meassage');
 
-$fail = Genome::InstrumentData::Command::Import::Basic->create(
-    analysis_project => $analysis_project,
-    library => $library,
-    source_files => \@source_files,
-    import_source_name => 'broad',
-    instrument_data_properties => [qw/ sequencing_platform=solexa lane=2 lane=3 flow_cell_id=XXXXXX /],
+throws_ok(
+    sub{
+        Genome::InstrumentData::Command::Import::Basic->execute(
+            analysis_project => $analysis_project,
+            library => $library,
+            source_files => [ $source_file ],
+            import_source_name => 'broad',
+            instrument_data_properties => [qw/ sequencing_platform=solexa lane=2 lane=3 flow_cell_id=XXXXXX /],
+        );
+    },
+    qr/Multiple values for instrument data property! lane => 2, 3/,
+    'Fails w/ invalid instrument_data_properties',
 );
-ok(!$fail->execute, 'Fails w/ invalid instrument_data_properties');
-is($fail->error_message, 'Failed to process instrument data properties!', 'Correct error meassage');
 
-$fail = Genome::InstrumentData::Command::Import::Basic->create(
-    analysis_project => $analysis_project,
-    library => $library,
-    source_files => \@source_files,
-    import_source_name => 'broad',
-    downsample_ratio => 0.25,
-    instrument_data_properties => [qw/ downsample_ratio=0.24 /],
+throws_ok(
+    sub{
+        Genome::InstrumentData::Command::Import::Basic->execute(
+            analysis_project => $analysis_project,
+            library => $library,
+            source_files => [ $source_file ],
+            import_source_name => 'broad',
+            downsample_ratio => 0.25,
+            instrument_data_properties => [qw/ downsample_ratio=0.24 /],
+        );
+    },
+    qr//,
+    'Fails w/ conflicting cmd and instdata properties instrument_data_properties',
```

Comment:
Not something you changed, but this says `instdata properties instrument_data_properties`.